.github/workflows/validate-pull-requests.yml

```
@@ -14,8 +14,19 @@ jobs:
         uses: actions/setup-dotnet@v3.0.3
         with:
           dotnet-version: 7.0.x
+      - name: Check docstool
+        run: |
+           curl https://www.myget.org/F/particular/api/v3/flatcontainer/particular.docstool/index.json >> docstool.version
```

This file contains all versions. Should I call it .versions?

The get request might slightly more expensive over time when we push more versions but this version is still many times faster than doing nuget list (that one took on average around 10-11 seconds)

---

How about this?
```suggestion
           curl https://www.myget.org/F/particular/api/v3/flatcontainer/particular.docstool/index.json >> docstool-versions.json
```

And at some point we should clean out some of those versions to conserve MyGet space given I can't think of a single reason to keep very old versions of DocsTool around.